stock_mts_mto_mrp/model/mrp_production.py

```diff
@@ -0,0 +1,37 @@
+# Copyright <2018> <Arturo Flores, Vauxoo>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+from odoo.exceptions import UserError
+
+
+class MrpProduction(models.Model):
+    """ Manufacturing Orders """
+    _inherit = 'mrp.production'
+
+    @api.multi
+    def _adjust_procure_method(self):
+        """This method is taken from odoo as is from the module mrp. This is
+        done because there was no way to interfiere in the decision making
+        about what routes to take"""
+        try:
+            mto_route = self.env['stock.warehouse']._get_mto_route()
+        except UserError:
+            mto_route = False
+        # This new route is for taking in count this module, the rest is the same as odoo # noqa
+        mto_route |= self.env['stock.location.route'].search([
```

Comment:
If there is a UserError exception then you will raise a error here of 
`False |= browse`

Better use:
```python
        try:
            mto_route = self.env['stock.warehouse']._get_mto_route()
        except UserError:
            mto_route = self.env['stock.warehouse']
```

Reply:
thanks, should I propose(later) a fix to the odoo too?